scripts/smeshr/prepare_HRRR_SC_inputs.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+# Script Name: prepare_HRRR_SC_inputs.sh
+# Author: J. Michelle Hu | University of Utah
+# Date: 9.24
+# Description: Generates directory structure and runs smeshr scripts accordingly
+# Requires smeshr conda env
+# Usage: ./prepare_HRRR_SC_inputs.sh <BASIN_NAME> <WY>
+# Example: prepare_HRRR_SC_inputs.sh animas 2022
+
+set -e
+echo ; echo "Running $0 $1 $2" ; echo "    from $(pwd)"
+
+# Script running flags
+verbose=true
+realrun=true
+
+echo ; echo verbose is $verbose, realrun is $realrun ; echo
+
+#==========================================================  
+#========  Establish variables and directory names ========
+#==========================================================  
+# Basin-specific variables
+BASIN=$1
+WY=$2
+YEAR=$(echo "${WY} - 1" | bc)
+
+# Model output directory - modify at will
+MODEL_DIR=/uufs/chpc.utah.edu/common/home/skiles-group1/jmhu/model_animas_only/
+
+# # Global directories
+# SMESHR_DIR=/uufs/chpc.utah.edu/common/home/skiles-group3/SMESHR/
+SMESHR_DIR=/uufs/chpc.utah.edu/common/home/skiles-group1/jmhu/smeshr_littleanimas_only/
+
+# Basin-specific output directories
+TCDC_out=${SMESHR_DIR}${BASIN}_hrrrsc/TCDC_out/ 
+DSWRF_DIR=${SMESHR_DIR}${BASIN}_hrrrsc/DSWRF/
+SMRF_DIR=${MODEL_DIR}/${BASIN}_100m_isnobal/wy${WY}/${BASIN}_basin/
+NET_SOLAR_DIR=${SMESHR_DIR}${BASIN}_hrrrsc/net_HRRR_SMRF/
+OUTDIR=${MODEL_DIR}${BASIN}_100m_isnobal_hrrrsc/wy${WY}/${BASIN}_basin_100m/
+
+# isnoda scripts and repo directory, edit ISNODA_DIR to your install
+ISNODA_DIR=/uufs/chpc.utah.edu/common/home/u6058223/git_dirs/isnoda/
+net_solar_script=${ISNODA_DIR}scripts/smeshr/SMRF/net_dswrf_for_topo_SMRF.sh
+linker_script=${ISNODA_DIR}scripts/smeshr/HRRR_linker.sh
+
+# #==========================================
+# #========== Prepare net_solar.nc ==========
+# #==========================================
+# # Prepare net_solar.nc from time decay SMRF albedo and HRRR SW
+# # Create directory for net solar output files
+# if $verbose ; then echo mkdir -v ${NET_SOLAR_DIR} ; echo ; fi
+# if $realrun ; then 
+#     if [ ! -d ${NET_SOLAR_DIR} ] ; then
+#         mkdir -v ${NET_SOLAR_DIR}
+#     fi
+# fi
+
+# if $verbose ; then 
+#     echo "${net_solar_script} "
+#     echo "    ${WY} "
+#     echo '    "0 1 2 3 4 5 6 7 8 9 10 11" '
+#     echo "    ${DSWRF_DIR} "
+#     echo "    ${SMRF_DIR} "
+#     echo "    ${NET_SOLAR_DIR}"
+# fi
+
+# if $realrun ; then 
+#     ${net_solar_script} ${WY} "0 1 2 3 4 5 6 7 8 9 10 11" ${DSWRF_DIR} ${SMRF_DIR} ${NET_SOLAR_DIR}
+# fi
```

Comment:
Seems that this should not be commented out in order to actually produce the files?

Reply:
Fixed in [957ee13](https://github.com/UofU-Cryosphere/isnoda/pull/64/commits/957ee13d3a96808287c904ced36e5727853b7f7c), thanks for the catch!